cirq/testing/random_circuit.py

```diff
@@ -72,15 +76,24 @@ def random_circuit(qubits: Union[Sequence[ops.Qid], int],
     if n_qubits < 1:
         raise ValueError('At least one qubit must be specified.')
 
+    if isinstance(random_state, np.random.RandomState):
+        prng = random_state
+    else:
+        prng = np.random.RandomState(random_state)
```

Comment:
```python
if random_state is None:
  prng = np.random
```

It's important not to change the default behavior of using the global random number generator. As @Strilanc  pointed out in #2112 , you could potentially get into a situation where repeated calls to `np.random.RandomState(None)` will give you less-than-random random states. For example, if the default seed is the time, if you call it fast enough you'd seed with the same values

Reply:
Good point. I'll fix this.